migration/sql-files/002-fix-273-workitemtype.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE work_item_types
+SET fields = '{"system.state": {"Type": {"Kind": "enum", "Values": ["new", "in progress", "resolved", "closed"], "BaseType": {"Kind": "string"}}, "Required": true}, "system.title": {"Type": {"Kind": "string"}, "Required": true}, "system.creator": {"Type": {"Kind": "user"}, "Required": true}, "system.assignee": {"Type": {"Kind": "user"}, "Required": false}, "system.description": {"Type": {"Kind": "string"}, "Required": false}}';
```

Comment:
Hi @aslakknutsen , 
This is the migration requirement which held up merging this to master in the previous sprint.

Ref: https://github.com/almighty/almighty-core/issues/273
We had to add the optional field `remote_item_id` and the state `open` ..